### DISCUSSION TITLE
Include self created to the connecting model ?

### DISCUSSION THREAD

**Discussion by nielpattin** [prisma/prisma#14372] [Q&A] [answered]
**The title quite sounds weird but this is what I want to make**

**I have 2 models**

```prisma
model User {
  id          Int       @id @default(autoincrement())
  name        String    @db.VarChar(255)
  email       String    @db.VarChar(255)
  avatarLink  String    @db.VarChar(255)
  ownerServer Server[]  @relation("OwnerServer")
  messages    Message[]
  servers     Server[]
}

model Server {
  id       Int       @id @default(autoincrement())
  name     String    @db.VarChar(255)
  owner    User      @relation(name: "OwnerServer", fields: [ownerId], references: [id])
  ownerId  Int
  users    User[]
  channels Channel[]
}
```

**The code**

```js
const server = await prisma.server.create({
      data: {
         name: name,
         owner: {
            connect: {
               id: Number(ownerId)
            }
         }
      }
   })
   return server
```

### What I want is that I want to include the server itself to the user server[]

**I'm read the docs but nothing shows up in my eye.
The only thing I can do rn is that making a user update to add the created server to the user server[]**

**Comment by austincrim** [comment, marked as answer]
Hey @salyndev0 👋,

This is not possible in Prisma today. My recommendation would be to issue two separate queries for now.

Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you think this is something we can improve.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
